python/foglamp/services/core/server.py

```diff
@@ -582,8 +585,10 @@ def _check_readings_table(cls, loop):
 
     @classmethod
     def _start_core(cls, loop=None):
-        _logger.info("start core")
+        if cls.running_in_safe_mode:
```

Comment:
An explanatory comment will be better to explain what behaviour is expected from core running in safe-mode.